spec/unit/installer/target_installer/pod_target_installer_spec.rb

```diff
@@ -398,6 +398,60 @@ module Pod
           end
         end
       end
+
+      describe 'concerning resource bundles' do
+        before do
+          config.sandbox.prepare
+
+          @project = Project.new(config.sandbox.project_path)
+
+          config.sandbox.project = @project
+
+          @spec = fixture_spec('banana-lib/BananaLib.podspec')
+          @spec.resource_bundle = { 'banana_bundle' => ['Resources/**/*'] }
```

Comment:
Here we should also set `resources` attribute to nil because current state will cause resources to be added multiple times to the project, which is not the case you want to test.